script/elevate-cpanel.PL

```diff
@@ -609,12 +620,30 @@ EOS
     say <<EOS;
 This is your chance now to cancel the update process.
 You can hit CTRL-C to abort, otherwise the install starts in a few seconds.
+
+It is *highly* recommended that you have a full backup or snapshot
+of your server before proceeding.
 EOS
 
     do { sleep 1; print '.' }
       for 1 .. 6;
     print "\n";
```

Comment:
Lines 628-630 should also be removed since we require the user to confirm now (unless they give the option to bypass the confirmation.)

Reply:
Makes sense.  Having a prompt eliminates the need for the six second delay.

Reply:
Could we move or copy 624-625 to the --help output?

Reply:
I imagine there would be no problem copying this over to --help.  I think we still want it here has well so they have the warning before answering the prompt.